system/jhtree/jhtree.ipp

```diff
@@ -167,37 +167,76 @@ public:
 
 class jhtree_decl CKeyCursor : public IKeyCursor, public CInterface
```

Comment:
minor: CInterfaceOf<IKeyCursor> would save 8 bytes per cursor.

Reply:
Will address in a future PR